src/NLog/Targets/TraceTarget.cs

```diff
@@ -66,8 +66,8 @@ public sealed class TraceTarget : TargetWithLayout
         /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
         /// </summary>
         /// <docgen category='Output Options' order='100' />
-        [DefaultValue(false)]
-        public bool RawWrite { get; set; }
+        [DefaultValue(true)]
+        public bool ForceTraceWriteLine { get; set; } = true;
```

Comment:
Think you need to keep `RawWrite` around as obsolete (Property that just assigns `ForceTraceWriteLine`)

Reply:
Maybe just keep the name RawWrite then.

Reply:
Like how `ForceTraceWriteLine` explains what is going to happen. And when making the breaking change of change the default-behavior (to use Trace WriteLine by default). Then might as well do it nicely.